crates/ruff_server/src/server/api/requests/code_action.rs

```diff
@@ -112,14 +112,14 @@ fn fix_all(snapshot: &DocumentSnapshot) -> crate::Result<CodeActionOrCommand> {
             None,
         )
     };
-    let action = types::CodeAction {
+
+    Ok(CodeActionOrCommand::CodeAction(types::CodeAction {
         title: format!("{DIAGNOSTIC_NAME}: Fix all auto-fixable problems"),
-        kind: Some(types::CodeActionKind::SOURCE_FIX_ALL),
-        edit,
-        data,
+        kind: Some(crate::SOURCE_FIX_ALL_RUFF),
+        edit: edit.clone(),
+        data: data.clone(),
         ..Default::default()
```

Comment:
The clones aren't necessary
```suggestion
        kind: Some(crate::SOURCE_FIX_ALL_RUFF),
        edit,
        data,
        ..Default::default()
```